benchmark/es/spread-bench.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const common = require('../common.js');
+const assert = require('assert');
+
+const bench = common.createBenchmark(main, {
+  method: ['apply', 'spread', 'call-spread'],
+  count: [5, 10, 20],
+  context: ['context', 'null'],
+  millions: [5]
+});
+
+function makeTest(count) {
+  return function test(...args) {
```

Comment:
Is it intentional to use spread here as well? It might be interesting to test with/without it in case V8 should make optimizations depending on use of spread in either/both places?